ddt does not expect flake8-2020, flake8-bugbear and flake8-docstrings plugins.

--- ddt-1.7.2/tox.ini.orig
+++ ddt-1.7.2/tox.ini
@@ -11,7 +11,7 @@
     PyYAML
 commands = 
     pytest --cov=ddt --cov-report html
-    flake8 ddt.py test
+    flake8 --extend-ignore=B,D,YTT ddt.py test
 
 [flake8]
 max-line-length = 127
